src/Core/src/Platform/Android/KeyboardExtensions.cs

```diff
@@ -37,37 +30,16 @@ public static InputTypes ToInputType(this Keyboard self)
 				var spellcheckEnabled = (custom.Flags & KeyboardFlags.Spellcheck) == KeyboardFlags.Spellcheck;
 				var suggestionsEnabled = (custom.Flags & KeyboardFlags.Suggestions) == KeyboardFlags.Suggestions;
 
-				if (!capitalizedSentenceEnabled && !spellcheckEnabled && !suggestionsEnabled)
```

Comment:
This block of "if" statements went through every possible combination of `capitalizedSentenceEnabled`, `spellcheckEnabled`, `suggestionsEnabled`. I simplified it by just turning on the appropiat flags.